diff --git a/vendor/knative.dev/hack/codegen-library.sh b/vendor/knative.dev/hack/codegen-library.sh
index 01a4313..5f8e530 100644
--- a/vendor/knative.dev/hack/codegen-library.sh
+++ b/vendor/knative.dev/hack/codegen-library.sh
@@ -19,7 +19,7 @@
 # Store Bash options
 oldstate="$(set +o)"
 
-set -Eeuo pipefail
+set -exvo pipefail
 
 export repodir kn_hack_dir kn_hack_library \
   MODULE_NAME CODEGEN_TMP_GOPATH CODEGEN_ORIGINAL_GOPATH GOPATH GOBIN \
